### PR TITLE
feature(ui-ux): added input selector

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/InputSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+          <div className="relative mt-1">
+            <Listbox.Button
+              onClick={(event) => {
+                if (disabled) {
+                  event.preventDefault();
+                }
+              }}
+              className={clsx(
+                "relative w-full outline-0",
+                disabled && "cursor-default",
```
Comment (kyleleow) [inline apps/web/src/components/InputSelector.tsx:75]: `cursor-pointer`

Comment (fullstackninja864) [reply apps/web/src/components/InputSelector.tsx:75]: for disabled input i have used `cursor-default` as we cant open selection

Comment (kyleleow) [reply apps/web/src/components/InputSelector.tsx:75]: i mean for non-disabled option

Comment (fullstackninja864) [reply apps/web/src/components/InputSelector.tsx:75]: its button, so its default cursor is pointer only

File: apps/web/src/components/InputSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                    data-testid={name}
+                    className="w-6 h-6 lg:w-9 lg:h-9"
+                  />
+                  <span className="block truncate text-dark-1000 ml-2 text-sm lg:text-xl">
+                    {name}
+                  </span>
+                </div>
+                {!disabled && (
+                  <span className="text-dark-900">
+                    <FaChevronDown
```
Comment (kyleleow) [inline apps/web/src/components/InputSelector.tsx:102]: the icon stroke width looks slightly thicker than on figma, can we reduce it?

Comment (fullstackninja864) [reply apps/web/src/components/InputSelector.tsx]: updated icon

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**